scripts/agent-evals/src/runner/gemini-cli-runner.ts

```diff
@@ -34,10 +36,17 @@ interface ParsedTelemetryLog {
   }[];
 }
 
+interface CheckResult {
+  success: boolean;
+  messages: string[];
+}
+
 export class GeminiCliRunner implements AgentTestRunner {
   private readonly cli: InteractiveCLI;
   private readonly telemetryPath: string;
   private readonly telemetryTimeout = 15000;
+  readonly runDir: string;
+  private readonly userDir: string;
```

Comment:
Instead of these, you can do `private readonly dirs: RunDirectories,` in the constructor and use dirs.runDir and dirs.userDir

Reply:
Ah yeah then you can just do `readonly dirs:...`